src/gsw_internal_funcs/mod.rs

```diff
@@ -1105,7 +1119,7 @@ fn gibbs_ice(nt: u8, np: u8, t: f64, p: f64) -> Result<f64> {
     );
 
         let ans = (g0 - TT*(s0*tau - g.re)).re;
-        return Ok(ans);
+        Ok(ans)
```

Comment:
Since `ans` is not re-used, let's reduce the code straight to:

`Ok(g0 - TT*(s0*tau - g.re)).re)`

The main argument against that would be to help verify the equation, but it seems small enough to be easy to check, what do you think?

Reply:
Yes, that makes sense. I just used ans for debugging, but it would be simpler just to return the calculation.